src/nupic/algorithms/KNNClassifier.py

```diff
@@ -1020,6 +1067,12 @@ def getAdaptiveSVDDims(self, singularValues, fractionOfMax=0.001):
 
 
   def finalizeSVD(self, numSVDDims=None):
+    """
+    TODO: Somebody help me write this.
+
+    :param numSVDDims:
+    :return:
+    """
```

Comment:
@numenta/nupic-reviewers I need help writing this. You can simply respond in a comment with a function description and I'll do the rest.